roles/serviceassurance/defaults/main.yml

```diff
@@ -2,11 +2,15 @@
 # set default enablement
 metrics_enabled: true
 events_enabled: false
+highAvailabilityEnabled: false
 
 # default values
 prometheus_storage_class: ""
 prometheus_storage_resources: {}
 prometheus_storage_selector: {}
 prometheus_pvc_storage_request: 20G
 
+# defaults for when high availbilty is enabled
+elasticSearchNodeCount: 3
```

Comment:
`elasticsearch_node_count` -- we would expect the inbound camelCase to be `elasticsearchNodeCount`